guides/_subsections/add-and-edit-recording-metadata-labels.md

```diff
@@ -10,72 +10,72 @@ Users can attach metadata or custom labels to JDK flight recordings managed by C
   </li>
   <li>
     {% include howto_step.html
-      summary="Add metadata labels to the Create Recording form"
+      summary="Add <code>Metadata Labels</code> to the <i>Create Recording</i> form"
       image-name="2.3.0/add-and-edit-recording-metadata-labels-1.png"
       text="
-        When creating a custom flight recording with Cryostat, expand the form section <i>Show metadata options</i>. Click <i>Add Label</i> and add key-value label pairs to the form. Finally, click <i>Create</i> to attach the labels and create the recording.
+        When creating a custom <code>Flight Recording</code> with <b>Cryostat</b>, expand the form section <i>Show metadata options</i>. Click <i>Add Label</i> and add <i>key-Value</i> <code>Label</code> pairs to the form. Finally, click <i>Create</i> to attach the <code>Labels</code> and create the <code>Recording</code>.
       "
     %}
   </li>
   <li>
     {% include howto_step.html
-      summary="View your labels on the Active Recordings Table"
+      summary="View your <code>Labels</code> on the <i>Active Recordings Table</i>"
```

Comment:
You should go through all these files and make the "tables" and "views" etc. consistent. In some of these files, like this one, table is italicized along with the phrase before it. However, in other files, the word "table" or "view" is not capitalized or italicized.